src/js/view/utils/views-manager.js

```diff
@@ -37,27 +35,6 @@ function matchIntersection(entry, group) {
     }
 }
 
-function scheduleResponsiveRedraw() {
```

Comment:
Could we do something where the ResizeListener's callbacks were debounced calls to this method?